addon/ActivityStreams.js

```diff
@@ -105,12 +104,8 @@ function ActivityStreams(metadataStore, options = {}) {
   );
   this._experimentProvider.init();
 
-  this._tabTracker = new TabTracker(
-    this.appURLs,
-    options.clientID,
-    this._memoized,
-    this._experimentProvider.experimentId
-  );
+  this._tabTracker = tabTracker;
+  this._tabTracker.init(this.appURLs, this._memoized, this._experimentProvider.experimentId);
```

Comment:
when we create the shield study we can change tab tracker to not depend on the memoizer or the experiment provider but for now this is a fine change, it just needs to be available out of the scope of activity stream